lib/commons/builder/config.rb

```diff
@@ -15,4 +14,18 @@ def initialize(file)
     )
   end
 
+  def languages
+    @languages ||= begin
```

Comment:
I think I generally wouldn't bother caching the results of these two methods, since they're doing very little work. That would also meant that you could rewrite this more simply as:

```ruby
    return values[:languages] if values[:langages]
    puts "WARNING: config.json should now use a list of language codes"
    values[:language_map].values
```

... at the cost of having that warning emitted on every call. (You don't need to change this if you don't want to - this is large a "for interest" comment.)

Reply:
Yeah, I was trying to avoid the warning on every use thing, which is why I wanted `languages` cached :-)